index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Template Site 1</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="container">
+    <div id="section">
+      <div id="left">
+        <h6>Looking for a <span>solution</span> <br> for your business?</h6>
+        <p>Check out Sogolytics enterprise feedback and <br> experience management platform.</p>
+        <div id="a-section">
+          <a href="#" id="first" class="link">Learn more</a>
+          <a href="#" id="second" class="link">Schedule a consultation</a>
+        </div>
+      </div>
+      <div id="right">
```

Comment:
This can be called image-wrapper instead of right or left div. Again uses classes instead of ID.